aimmo-game/simulation/map_generator.py

```diff
@@ -13,47 +13,52 @@
 from simulation.world_map import WorldMapStaticSpawnDecorator
 from simulation.world_map import DEFAULT_LEVEL_SETTINGS
 
-LOGGER = logging.getLogger(__name__)
-
-
-class _BaseGenerator(object):
-    __metaclass__ = abc.ABCMeta
-
-    def __init__(self, settings):
-        self.settings = settings
+from simulation.custom_map import BaseGenerator
+from simulation.custom_map import BaseLevelGenerator
+from simulation.custom_map import EmptyMapGenerator
+from simulation.cell import *
 
-    def get_game_state(self, avatar_manager):
-        return GameState(self.get_map(), avatar_manager, self.check_complete)
+LOGGER = logging.getLogger(__name__)
 
```

Comment:
you get the idea :)

Reply:
Just to check, two lines please https://www.python.org/dev/peps/pep-0008/#blank-lines?

Reply:
yes :D